cluecumber-builder/pom.xml

```diff
@@ -31,33 +25,21 @@
         </developer>
     </developers>
 
-    <licenses>
-        <license>
-            <name>Apache License, Version 2.0</name>
-            <url>https://www.apache.org/licenses/LICENSE-2.0.txt</url>
-            <distribution>repo</distribution>
-            <comments>A business-friendly OSS license</comments>
-        </license>
-    </licenses>
+    <organization>
+        <name>trivago N.V.</name>
+        <url>http://www.trivago.de</url>
+    </organization>
 
-    <packaging>maven-plugin</packaging>
+    <groupId>com.trivago.rta</groupId>
+    <artifactId>cluecumber-builder</artifactId>
```

Comment:
`cluecumber-builder` is kind of a strange name. What is the purpose of this?

Reply:
This is where I can build the maven task and the core together to avoid having to manually build each of the projects maven.